tests/PHPUnit/System/expected/test_OneVisitorTwoVisits__DevicesDetection.getType_day.xml

```diff
@@ -37,6 +37,12 @@
 		<logo>plugins/DevicesDetection/images/screens/mobile.gif</logo>
 	</row>
 	<row>
+		<label>Phablet</label>
+		<nb_visits>0</nb_visits>
+		<segment>deviceType==phablet</segment>
+		<logo>plugins/DevicesDetection/images/screens/unknown.gif</logo>
```

Comment:
there would be one for mobile, tablet and car but I don't think we would find everything in font awesome. Btw. I think it's difficult to find good icons that show a visible different between tablet and phablet...